instrumentation/compose/src/main/java/de/mannodermaus/junit5/compose/AndroidComposeExtension.kt

```diff
@@ -46,6 +46,11 @@ public inline fun <reified A : ComponentActivity> createAndroidComposeExtension(
     return createAndroidComposeExtension(A::class.java)
 }
 
+@ExperimentalTestApi
+public inline fun <reified A : ComponentActivity> createAndroidComposeExtension(noinline scenarioSupplier: () -> ActivityScenario<A>): AndroidComposeExtension<A> {
+    return createAndroidComposeExtension(A::class.java, scenarioSupplier)
+}
+
```

Comment:
Please add a Javadoc to this! You can copy from the overloaded method and edit the text as necessary.